.github/actions/checkout_target_commit/action.yml

```diff
@@ -1,81 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
We do not need that any more as we do not have `pull_request_target` workflow any more